src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -535,6 +535,8 @@ def load_arguments(self, _):
         c.argument('skip_resource_name_params', action='store_true')
         c.argument('skip_all_params', action='store_true')
         c.argument('resource_ids', nargs='+', options_list='--resource-ids')
+        c.argument('output_format', options_list=['--output-format'],
+                    help="The format of the exported template. Allowed values: 'json', 'arm', 'bicep'. Defaults to 'json'.")
```

Comment:
Because this parameter is similar to the global parameter `--output` and can be confusing, it is recommended to change the parameter name to `export_format`